main.js

```diff
@@ -52,38 +68,31 @@ function createWindow() {
     mainWindow = null
   });
 
-  globalShortcut.register('mediaplaypause', function () {
-    // When the playpause function key is pressed, toggle playback by
-    // using Pandora's spacebar shortcut.
-    mainWindow.webContents.sendInputEvent({
-      type: "keyDown",
-      keyCode: "\u0020"
-    });
-    mainWindow.webContents.sendInputEvent({
-      type: "keyUp",
-      keyCode: "\u0020"
-    });
+  // ⏯ Toggle play/pause with space shortcut
+  globalShortcut.register('MediaPlayPause', () => pressShortcut('Space'));
+
+  // ⏩ Skip to next track with right arrow shortcut
+  globalShortcut.register('MediaNextTrack', () => pressShortcut('Right'));
+
+  // ⏪ Mark song as "liked" (if it is not already)
```

Comment:
Pandora already has +/- keyboard shortcuts. Let’s just follow that pattern and use `CmdOrCtrl+Alt++`/`CmdOrCtrl+Alt+-` if possible.